spec/addressable/template_spec.rb

```diff
@@ -91,7 +91,7 @@
     '{hello}' => '1234',
     '{nothing}' => '',
     '{sym}' => 'symbolic',
-    '{decimal}' => '0.1E1'
+    '{decimal}' => RUBY_VERSION < '2.4.0' ? '0.1E1' : '0.1e1'
```

Comment:
I'd like to use single quotes in this PR, because it is difficult to understand the difference with existing code.